lib/cocoapods/installer/analyzer.rb

```diff
@@ -217,7 +217,7 @@ def generate_podfile_state
       def update_repositories
         sources.each do |source|
           if source.git?
-            config.sources_manager.update(source.name)
+            config.sources_manager.update(source.name, true)
```

Comment:
@adellibovi I'd be fine with just passing true for now... far friendlier for most users I think.